tests/test_operations.py

```diff
@@ -423,6 +424,78 @@ def paginate(self):
                     ),
                 )
 
+    def test_ssmparams_crud(self):
+        # CRUD (create read update delete) test for setting environment variables in SSM param store
+        def _wrap_ssm_env(e):
+            # Package up the SSM environment the way AWS returns it
+            # Value should be a string of JSON data
+            ssm_e = {"Parameter": {"Name": "environment", "Value": json.dumps(e)}}
+            return ssm_e
+
+        testvar_name = fix_ssm_variable_prefix(random_alphanumeric_string())
+        testvar_value = "Hello world!"
+        testvar_value2 = "Goodbye world!"
+
+        # Assemble an old and new environment to return
+        old_env = {"dummy_key": "dummy_value"}
+        new_env = dict(**old_env)
+        new_env[testvar_name] = testvar_value
+        ssm_old_env = _wrap_ssm_env(old_env)
+        ssm_new_env = _wrap_ssm_env(new_env)
+
+        with self.subTest("Create a new SSM parameter"):
```

Comment:
Nit: to reduce nesting, these context managers can all be invoked in the same `with` statement a la:

    with x() as ctx, \
         y() as ctx2, \
         z() as ctx3(():

etc.

Reply:
Very nice!